ga4gh/datamodel/reads.py

```diff
@@ -110,22 +109,22 @@ def __init__(self, id_):
         self._readGroups.append(readGroup)
 
 
-class HtslibReadGroupSet(datamodel.PysamSanitizer, AbstractReadGroupSet):
+class HtslibReadGroupSet(AbstractReadGroupSet, datamodel.PysamDatamodelMixin):
```

Comment:
reversing this order is bad.  the classes are essentially checked in left to right order for method resolution order (MRO).  the true "base class" should always go last.